tests/jobs/test_backfill_job.py

```diff
@@ -82,6 +83,7 @@ def setUpClass(cls):
     def setUp(self):
         clear_db_runs()
         clear_db_pools()
+        add_default_pool_if_not_exists()
```

Comment:
Could you update clear_db_pools instead to not delete the default pool?

Reply:
updated